perf/lua/column_scan_module.c

```diff
@@ -146,7 +145,6 @@ sum_scanner_rv_lua_func(struct lua_State *L)
 	if (rv == NULL)
 		return luaL_error(L, "run init() first");
 	uint32_t space_id = luaL_checkinteger(L, 1);
-	uint32_t index_id = luaL_checkinteger(L, 2);
```

Comment:
Instead of dropping this line, let's pass `index_id` to `box_raw_read_view_index_by_id` instead of 0.

Reply:
Done.